packages/core/src/components/dialog/dialogFooter.tsx

```diff
@@ -52,35 +52,15 @@ export interface DialogFooterProps extends Props {
  *
  * @see https://blueprintjs.com/docs/#core/components/dialog.dialog-footer-props
  */
-export class DialogFooter extends AbstractPureComponent<DialogFooterProps> {
-    public static defaultProps: DialogFooterProps = {
-        minimal: false,
-    };
-
-    public render() {
-        return (
-            <div
-                className={classNames(Classes.DIALOG_FOOTER, this.props.className, {
-                    [Classes.DIALOG_FOOTER_FIXED]: !this.props.minimal,
-                })}
-            >
-                {this.renderMainSection()}
-                {this.maybeRenderActionsSection()}
-            </div>
-        );
-    }
-
-    /** Render the main footer section (left aligned). */
-    private renderMainSection() {
-        return <div className={Classes.DIALOG_FOOTER_MAIN_SECTION}>{this.props.children}</div>;
-    }
-
-    /** Optionally render the footer actions (right aligned). */
-    private maybeRenderActionsSection() {
-        const { actions } = this.props;
-        if (actions == null) {
-            return undefined;
-        }
-        return <div className={Classes.DIALOG_FOOTER_ACTIONS}>{actions}</div>;
-    }
-}
+export const DialogFooter: React.FC<DialogFooterProps> = ({ children, actions, minimal = false, className }) => {
+    return (
+        <div
+            className={classNames(Classes.DIALOG_FOOTER, className, {
+                [Classes.DIALOG_FOOTER_FIXED]: !minimal,
+            })}
+        >
+            <div className={Classes.DIALOG_FOOTER_MAIN_SECTION}>{children}</div>
+            {actions && <div className={Classes.DIALOG_FOOTER_ACTIONS}>{actions}</div>}
+        </div>
+    );
+};
```

Comment:
Can we add `displayName` here?
```suggestion
};
DialogFooter.displayName = `${DISPLAYNAME_PREFIX}.DialogFooter`;
```